best-practices.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+    env: {
+        browser: true,
+        node: true
+    },
+    rules: {
+        // 'accessor-pairs': 2,
+        // 'array-callback-return': 2,
+        // 'block-scoped-var': 2,
+        // NO_RULE 'class-methods-use-this': 2,
```

Comment:
Had a change of heart and completely changed the direction.

Now we have
- `./configurations` all configurations every application should ideally pick from here
- `./rules` rules grouped by plugin type 
- `./rules/eslint` eslint rules split by categories
- `./rules/mocha` relevant mocha rules

So you could use it like this:

By default it is es5 config

```js
{
    "extends": "casumo"
}
```

or

```js
{
    "extends": "casumo/configurations/es5-browser"
}
```

and in the gulp folder

```js
{
    "extends": "casumo/configurations/es5-node"
}
```

Also all rules are enabled (no commented rules), this is to prepare for the check for missing rules npm script